src/commands/index.js

```diff
@@ -5,6 +5,10 @@ var previews = require("../previews"); //eslint-disable-line
 module.exports = function(client) {
   var loadCommand = function(name) {
     var cmd = require("./" + name);
+    // A hack for loading .ts commands
```

Comment:
It's not a hack. "`.ts` commands export at `.default`" is an appropriate comment.

Reply:
Done